src/controllers/campaign/stepTwo.ts

```diff
@@ -0,0 +1,40 @@
+import { AppResponse } from '@/common/utils';
+import AppError from '@/common/utils/appError';
+import { catchAsync } from '@/middlewares';
+import { campaignModel as campaign } from '@/models/campaignModel';
+import { Request, Response } from 'express';
+
+const secondStep = catchAsync(async (req: Request, res: Response) => {
+	const { user } = req;
+	const { title, fundraiser, goal } = req.body;
+	const { campaignId } = req.params;
```

Comment:
Campaign ID will be in the req body not in params, only step would be be sent as param, also verify that the campaign to be updated was created by the logged in user!